modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1682,6 +1682,11 @@
 			<description name="Replaces the old methods of class `CommerceCountryService` with the new equivalents in the `CountryService` class." />
 			<weight>1</weight>
 		</check>
+		<check name="UpgradeJavaCommerceDestinationNamesCheck">
+			<category name="Upgrade" />
+			<description name="CommerceDestinationNames constants was replaced by DestinationNames" />
```

Comment:
@michaelmdelcavalcanti, please rename it to: "CommerceDestinationNames deprecated constants were replaced by DestinationNames' constants"

Reply:
Done.